<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Jade Cocoon Randomizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileClose"/>
    <addaction name="separator"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpGitHub"/>
    <addaction name="actionHelpDiscord"/>
    <addaction name="actionHelpJadeCocoonWiki"/>
    <addaction name="actionHelpChangelog"/>
    <addaction name="separator"/>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuSettingsTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionSettingsThemeDark"/>
     <addaction name="actionSettingsThemeLight"/>
    </widget>
    <addaction name="menuSettingsTheme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset theme="Open"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="icon">
    <iconset theme="Close"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="icon">
    <iconset theme="SaveAs"/>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="icon">
    <iconset theme="Exit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionHelpGitHub">
   <property name="icon">
    <iconset theme="GitHub"/>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionHelpDiscord">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/Discord.png</normaloff>:/icons/Discord.png</iconset>
   </property>
   <property name="text">
    <string>Discord</string>
   </property>
  </action>
  <action name="actionHelpJadeCocoonWiki">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/JcWiki.png</normaloff>:/icons/JcWiki.png</iconset>
   </property>
   <property name="text">
    <string>Jade Cocoon Wiki</string>
   </property>
  </action>
  <action name="actionHelpChangelog">
   <property name="icon">
    <iconset theme="Changelog"/>
   </property>
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/SwIcon.png</normaloff>:/icons/SwIcon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
  <action name="actionSettingsThemeDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionSettingsThemeLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
